Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="bondziks@gmail.com"
+
+ENV PYTHONNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The 'WORKDIR' should be an absolute path. Consider changing it to '/app' instead of 'app/'. This ensures that the working directory is correctly set within the container.